docs/running_psalm/issues.md

```diff
@@ -5,8 +5,6 @@
  - [ArgumentTypeCoercion](issues/ArgumentTypeCoercion.md)
  - [AssignmentToVoid](issues/AssignmentToVoid.md)
  - [CircularReference](issues/CircularReference.md)
- - [ComplexFunction](issues/ComplexFunction.md)
- - [ComplexMethod](issues/ComplexMethod.md)
```

Comment:
Both seem to be thrown.

Reply:
Well...perhaps we need to add documentation for those.